lib/ansible/modules/windows/win_get_url.py

```diff
@@ -57,49 +55,53 @@
         resource, so for this to work, the remote web server must support
         HEAD requests.
     version_added: "2.0"
-    required: false
-    choices: [ "yes", "no" ]
-    default: yes
+    type: bool
+    default: 'yes'
   username:
     description:
       - Basic authentication username
-    required: false
-    default: null
   password:
     description:
       - Basic authentication password
-    required: false
-    default: null
   skip_certificate_validation:
     description:
     - This option is deprecated since v2.4, please use C(validate_certs) instead.
     - If C(yes), SSL certificates will not be validated. This should only be used
       on personally controlled sites using self-signed certificates.
-    default: 'no'
     type: bool
+    default: 'no'
   validate_certs:
     description:
     - If C(no), SSL certificates will not be validated. This should only be used
       on personally controlled sites using self-signed certificates.
     - If C(skip_certificate_validation) was set, it overrides this option.
```

Comment:
Well, the reason I do it like this is because `validate_certs` defaults to `$true` and this is how we want it. I don't want to have a default for `skip_certificate_validation` because I want to know whether it was set deliberately (testing for `$null`).

So this is for convenience. At some point we will deprecate `skip_certificate_validation` and by that time this is moot anyway IMO.